search/binarySearch.py

```diff
@@ -0,0 +1,46 @@
+# O(lb(n))
+
+# While-loop version
+def biSearch (List, target):
```

Comment:
according to python conventions names of variables and functions should have the following format: lower_with_under. Names of variables should start with lowercase letters, names of classes with uppercase letters, more on this here: https://www.python.org/dev/peps/pep-0008/#prescriptive-naming-conventions